### ISSUE TITLE
Add grammatical error corrections

### ISSUE THREAD

**Issue by npm1** [GoogleChrome/chromium-dashboard#879]
It would be nice if the fields in chromestatus would show correction suggestions that are close to what one would get in Google Docs (which is where I used to write the fields, and is meant to be made deprecated) or Gmail (which is where the email to be sent to blink-dev is going to be written). Otherwise, any corrections suggested afterwards will need to be applied twice: in the place where such corrections were suggested and in the chromestatus. This is very inefficient.

**Comment by jpmedley** [comment]
I agree with the general tenor of this. You should know that we're trying to build a system where the duplication you mention isn't needed.
